src/app/app.component.html

```diff
@@ -27,13 +27,14 @@ <h3>
     <mat-checkbox cy-data="selectOnDrag" [(ngModel)]="selectOnDrag">Select on Drag</mat-checkbox>
     <mat-checkbox cy-data="disable" [(ngModel)]="disable">Disable</mat-checkbox>
     <mat-checkbox cy-data="selectMode" [(ngModel)]="selectMode">Select Mode</mat-checkbox>
+    <mat-checkbox cy-data="toggleWithoutShortcut" [(ngModel)]="toggleWithoutShortcut">Toggle on click without shortcut</mat-checkbox>
```

Comment:
Change text to `Select without Shortcut`. It's a little less but preserves the meaning I hope 😆